rust/src/ssh/ssh.rs

```diff
@@ -65,11 +65,12 @@ pub enum SSHEvent {
 
 #[repr(u8)]
 #[derive(Copy, Clone, PartialOrd, PartialEq, Eq)]
+/// cbindgen:prefix-with-name=true
```

Comment:
@jasonish any better trick ?

Reply:
Now I get in C `SSHConnectionState_BannerDone ` Ideally I would like `SshStateBannerDone`

Reply:
Elsewhere we just use full names inside the enum. Annoying to use from Rust though... So:
```
pub enum SSHConnectionState {
    SSHConnectionStateInProgress,
}
```
this is preferable if also using from C, as it keeps the identifier name the same.

See discussion here: https://github.com/OISF/suricata/pull/8417

Reply:
Sure I knew that, and that is the current state but this fits bad with the derive introduced here...

Should the derive do the magic to strip the common prefix ?

Reply:
Managed to have something with [Derive macro helper attributes](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macro-helper-attributes)